content/blog/2021-02-01-tidynhl-analyse/index.fr.md

```diff
@@ -0,0 +1,711 @@
+---
+title: "Repêchage de la LNH: est-ce vraiment une science inexacte?"
+slug: "tidy-nhl-draft-analysis"
+author: "Stéphane Caron"
+description: ""
+date: "2021-02-12"
+categories: ["Analytique"]
+type: post
+tags: 
+  - R
+  - Analytique de sports
+output:
+  html_document:
+    keep_md: yes
+featured: "hockey-draft-post.jpg"
+featuredpath: "img/headers/"
+reading_time: 10
+---
+
+
+
+**On entend souvent dire que le repêchage de la LNH, tout comme c'est le cas
+dans d'autres sports professionnels, est une science inexacte. J'imagine que cette
+expression fait référence au fait que si l'on doit repêcher un joueur $x$ à un moment 
+$t$ donné, il n'y a aucune certitude que ce joueur est le bon choix. Mais disons 
+qu'on prend le temps de regarder plusieurs choix, sur plusieurs années, est-ce toujours le 
+cas? Est-ce que le repêchage devient en quelque sorte une science plus "exacte" lorsque l'échantillon de joueurs repêchés devient plus grand?**
```

Comment:
si c'est un résumé peut-être utilisé
> commented

Reply:
J'trouve ça fait plus "quote" ou bien "phrase importante" non? Je le voyais plus comme un genre de chapeau, souvent mis en gras dans les articles?

Reply:
Allright then! Good for me.

Reply:
Hahah good 😊